data/newsletter/issue-385.markdown

```diff
@@ -4,12 +4,10 @@ This is a weekly summary of what's going on in its community.
 
 ## Featured
 
-undefined
+- [Celebrating the release of Amazonka 2.0](https://exploring-better-ways.bellroy.com/celebrating-the-release-of-amazonka-2-0.html) How [Bellroy](https://bellroy.com) thinks about open source
 
 ## Jobs
 
-undefined
-
 Trying to hire a Haskell developer?
 You should [advertise with us](https://haskellweekly.news/advertising.html)!
```

Comment:
I figured this was a better default than `undefined` but happy to be corrected on that